sew.lic

```diff
@@ -261,18 +249,16 @@ class Sew
 
   def do_sew(command, tool, lighten = false)
     waitrt?
-    crafting_magic_routine(@settings)
-    stow_item(left_hand) if checkleft
+    DRCA.crafting_magic_routine(@settings)
+    DRCC.stow_crafting_item(DRC.left_hand, @bag, @belt) unless DRC.left_hand == (tool)
     if @last_command == "cut my #{@mat_type} cloth with my scissors"
```

Comment:
Why do we store and check the `@last_command` at the top of `do_sew`, instead of doing this logic at the end of the previous execution of `do_sew`?

Reply:
I suppose it doesn't matter if that's how we're going to do it. An alternative would be checking feet slot? not sure how to do that. Could also put something like this in cleanup, instead of eatly in the process? Maybe extend the setup for both sewing and leather?